manifests/prometheus/prometheus-k8s-cm.yaml

```diff
@@ -42,6 +42,10 @@ data:
         source_labels: [__meta_kubernetes_service_name]
         target_label: job
       - action: replace
+        source_labels: [__meta_kubernetes_service_annotation_prometheus_io_job]
+        target_label: job
+        regex: (.+)
```

Comment:
Unfortunately the Prometheus default for regex is `(.*)`, which means if the annotation doesn't exist it would set job="".